system/CodeIgniter.php

```diff
@@ -446,7 +446,7 @@ protected function handleRequest(?RouteCollectionInterface $routes, Cache $cache
                 return $returnResponse ? $possibleResponse : $possibleResponse->pretend($this->useSafeOutput)->send();
             }
 
-            if ($possibleResponse instanceof Request) {
+            if ($possibleResponse instanceof IncomingRequest) {
```

Comment:
I believe it's possible for this to be CLIRequest also, isn't it?

Reply:
Oh, thanks! I was confused with spark.

Reply:
Fixed.